bitnami/tomcat/values.yaml

```diff
@@ -103,9 +103,15 @@ tomcatUsername: user
 ## ref: https://github.com/bitnami/containers/tree/main/bitnami/tomcat#creating-a-custom-user
 ##
 tomcatPassword: ""
-## @param existingSecret Use existing secret for password details (`tomcatPassword` will be ignored and picked up from this secret). The secret has to contain the key `tomcat-password`
+## @param existingSecret Use existing secret for credentials details (`tomcatUsername` and `tomcatPassword` will be ignored and picked up from this secret).
 ##
 existingSecret: ""
+## @param secretKeys.adminUsernameKey Name of key in existing secret to use for Tomcat admin username (overrides `tomcatUsername`). Only used when `existingSecret` is set.
+## @param secretKeys.adminPasswordKey Name of key in existing secret to use for Tomcat admin password (overrides `tomcatPassword`). Only used when `existingSecret` is set.
```

Comment:
I will let know in the comment that defaults keys `tomcat-username` and `tomcat-password` will be used. Just in case a user sets `existingSecret` and not `secretKeys`.